### PR TITLE
add: async support

### PR DESCRIPTION
**Pull request by MioYvo**
Add async support, based on [httpx](https://github.com/encode/httpx)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,68 @@ print(txn)
 print(txn.wait())
 # > {'id': '5182b96bc0d74f416d6ba8e22380e5920d8627f8fb5ef5a6a11d4df030459132', 'blockNumber': 6415370, 'blockTimeStamp': 1591951155000, 'contractResult': [''], 'receipt': {'net_usage': 283}}
 ```
+
+#### async
```
Comment (andelf) [inline README.md:31]: use H3 level heading

**Review by andelf — APPROVED** [review]
Good job.

**Comment by andelf** [comment]
🥳 merged. When docs are ready, the next release will be v0.2.0.

**Comment by MioYvo** [comment]
>  merged. When docs are ready, the next release will be v0.2.0.

😊 What do you need me to do about docs, I prefer to release v0.2.0 soon.

**Comment by andelf** [comment]
published as v0.2.0.
with minor doc updates.

**Pull request merged**